pkgs/applications/office/sleek/default.nix

```diff
@@ -0,0 +1,78 @@
+{ fetchFromGitHub
+, mkYarnPackage
+, fetchurl
+, fpm
+, wrapGAppsHook
+, lib
+}:
+
+mkYarnPackage rec {
+  pname = "sleek";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "ransome1";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-j5T4StCcHtNHEn0VycPJlX/w+F0jZYBPEgHI/n5nGiI=";
+  };
+
+  electron = fetchurl {
+    url = "https://github.com/electron/electron/releases/download/v18.3.7/electron-v18.3.7-linux-x64.zip";
+    hash = "sha256-4JjmZB6HWjBZhRiwtFDMKsG8/nfvK/1jkvIxJU+DUos=";
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook ];
+
+  patches = [
+    ./package.json.patch
+    ./snap/gui/sleek.desktop.patch
```

Comment:
```suggestion
    ./sleek.desktop.patch
```

Reply:
@auroraanna didn't properly read the first line of the patch and thought the folder it's in matters.